tests/functional/test_project.py

```diff
@@ -45,6 +46,12 @@ def test_extract_manifest(dependency_config, project_manager):
     assert type(manifest) == PackageManifest
 
 
+def test_meta(dependency_config, project_manager):
```

Comment:
Instead of using `dependency_config`, use the `temp_config` fixture to make your own meta config:

def test_meta(temp_config, project_manager):
  meta_config = {"meta": {"authors": ["Test Testerson"]}}
  with temp_config(meta_config):
      assert project_amager.meta.authors == ["Test Testerson"]
```

Reply:
Expand on my example a little, add the other properties and stuff